.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "env": {
+    "browser": true,
```

Comment:
We don't use browser env here, you can remove it

Reply:
removed